package.json

```diff
@@ -9,16 +9,24 @@
     "add:resource:immutable": "NODE_ENV=production npm install && cp -a ./node_modules/immutable ./Resources/Public/Javascript",
     "build:extension": "npm run add:resource:ckeditor && npm run add:resource:toastr && npm run add:resource:immutable && npm run build:css",
     "build:css": "npm run lint:css --silent && node-sass Resources/Private/Styles/ -o Resources/Public/Styles/",
+    "build:js": "./node_modules/.bin/browserify Resources/Private/Javascript/client.js -o Resources/Public/Javascript/Bundle.js -t [ babelify --presets [ es2015 react ] ]",
```

Comment:
You can run only browserify as npm understands the path. Then also add a section about it to the readme.